cfe_internal/enterprise/templates/httpd.conf.mustache

```diff
@@ -280,12 +280,6 @@ AddType    application/x-httpd-php-source php{{{vars.cfe_internal_hub_vars.php_v
   <IfModule rewrite_module>
     RewriteEngine On
 
-    {{^classes.cfe_enterprise_disable_http_redirect_to_https}}
-    # Force https with redirection
-    RewriteCond %{HTTPS} off
-    RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI} [R=301,L]
-    {{/classes.cfe_enterprise_disable_http_redirect_to_https}}
-
```

Comment:
Apache is no longer responsible for redirecting http -> https. Now, the class that used to be used to manage the Apache config section is instead used to manage a flag file.